src/app/containers/ATIAnalytics/ArticleAtiParams.test.jsx

```diff
@@ -39,12 +39,11 @@ describe('ArticleAtiParams', () => {
     atiAnalyticsAppName: 'news',
   };
   const requestContextStub = {
-    isUK: true,
-    origin: 'https://www.bbc.co.uk',
+    id: 'c0000000000o',
```

Comment:
The order of the props change depending on the file - could you please ensure they're alphabetically sorted? 
I will separately raise an issue to add the lint rule https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md to our config so that it is mentioned prior to committing in future.

Reply:
Done :)